userspace/libsinsp/container_engine/docker/async_source.cpp

```diff
@@ -795,3 +867,82 @@ bool docker_async_source::parse_docker(const docker_lookup_request& request, sin
 	return true;
 }
 
+
+void docker_async_source::compute_capabilities(sinsp_container_info& container, const Json::Value& add, const Json::Value& drop)
+{
+	vector<string> added_capabilities;
+	vector<string> dropped_capabilities;
+	bool all_added = false;
+	bool all_dropped = false;
+
+	for(const auto& cap : add)
+	{
+		string c = cap.asString();
+		if(parse_capability(c))
+		{
+			all_added = true;
+			break;
+		};
+		added_capabilities.push_back(c);
+	}
+	for(const auto& cap : drop)
+	{
+		string c = cap.asString();
+		if(parse_capability(c))
+		{
+			all_dropped = true;
+			break;
+		};
```

Comment:
`;`...typo? Same goes for the above one!

Reply:
Fixed now, thanks!